docs/content/docs/build/data-modeling.md

```diff
@@ -62,10 +61,10 @@ const schema = {
     dynamicObjectTypes: [ SharedCell, SharedMap ],
 }
 
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
+const { container, services } = await client.getContainer(/*service config*/, schema);
```

Comment:
I think the config here will just be the id string now, here and in the other occurrences below

Reply:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```